scripts/zones/Horlais_Peak/mobs/Derakbak_of_Clan_Wolf.lua

```diff
@@ -0,0 +1,28 @@
+-----------------------------------
+-- Area: Horlais Peak
+--  MOB: Derakbak of Clan Wolf
+-----------------------------------
+
+require("scripts/globals/missions");
+
+function onMobInitialize(mob)
+    mob:setMobMod(MOBMOD_SCRIPTED_2HOUR, 1);
+end;
+
+function onMobSpawn(mob)
+    mob:setLocalVar("2HOUR_HPP", math.random(35,60));
+end;
+
+function onMobFight(mob, target)
+    if (mob:getLocalVar("2HOUR_USED") == 0 and mob:getHPP() <= mob:getLocalVar("2HOUR_HPP") and  not mob:hasPet()) then
+        mob:setLocalVar("2HOUR_USED", 1);
+        mob:useMobAbility(732);
```

Comment:
and here https://github.com/DarkstarProject/darkstar/blob/master/scripts/globals/status.lua#L2013